_sources/configs/robots.php

```diff
@@ -0,0 +1,35 @@
+<?php
+# This file pretends to be a /robots.txt file (via Apache rewrite, see configs/mediawiki.conf)
+# Sitemap URL pattern is ${SITE_SERVER}${SCRIPT_PATH}/sitemap${SITEMAP_DIR}/sitemap-index-${MW_SITEMAP_IDENTIFIER}.xml
+
+ini_set( 'display_errors', 0 );
+error_reporting( 0 );
+
+if ( getenv( 'MW_ENABLE_SITEMAP_GENERATOR') == 'true' ) {
```

Comment:
We use isTrue() function to check this. The sitemap will be generated when MW_ENABLE_SITEMAP_GENERATOR value is one of "True" | "TRUE" | "true" | "1", but the sitemap will be added when  MW_ENABLE_SITEMAP_GENERATOR is 'true' only.
I would reuse the existing function, but any fix is welcome.

Reply:
IsTrue is a bash function https://github.com/WikiTeq/Taqasta/blob/master/_sources/scripts/functions.sh#L6 so we can't re-use it here, I have updated the comparison code to make it align